proposals/0167-app-services.md

```diff
@@ -698,6 +779,8 @@ If the app service consumer subscribed to updates from the app service provider
 
 ###### OnAppServiceData
 
+Notifications will come through as data changes. Only the data that changed should be expected to be included. This prevents the provider and subscriber traffic from becoming too large. However, it is recommended that these be batched together when possible as to avoid constant traffic.
```

Comment:
Update to:
However, it is recommended that these be batched together when **possible to avoid** constant traffic.